app/controllers/sessions_controller.rb

```diff
@@ -217,7 +218,7 @@ def destroy
 
   # This is an official action so that fine_print can check to see if terms need to be signed
   def redirect_back
-    super
+    super # Rails defines this
```

Comment:
What does `Rails defines this` tell us here?

Reply:
I could not find where `redirect_back` was defined, for the life of me. I think I eventually searched the Rails source code on github and though it would be a good idea to add a comment here. The comment above "This is an official action..." (to me) did not mean anything or I didn't understand what "official action" meant.

Reply:
Fixed; improved comment.